pkg/util/gotemplate.go

```diff
@@ -0,0 +1,31 @@
+package gotemplate
```

Comment:
Should we rename the directory `pkg/util` to `pkg/gotemplate`?

Reply:
Is pkg effectively "util" for you? I agree that "util" is a dirty word since it doesn't have clear encapsulation, it's a catch-all. Perhaps we can discuss over chat.